scripts/generate/templates/SafeCast.js

```diff
@@ -1,8 +1,63 @@
+const { assert } = require('chai');
```

Comment:
It doesn't make sense to use Chai here because there's a native assert module. Chai is a testing library. This change should do:

```suggestion
const assert = require('assert');
```